Loader related corrections for full-pipeline tests (#580)
DiamondLightSource/httomo: docs/source/pipelines/yaml.rst

DLS-specific templates
----------------------

> Those pipelines bellow can be specific to Diamond Light Source processing strategies and can vary between different tomographic beamlines. 

Reviewer: Slightly rewording:
```suggestion
These pipelines are specific to Diamond Light Source processing strategies and can vary between different tomographic beamlines. 
```
